horreum-web/src/utils.ts

```diff
@@ -152,3 +152,29 @@ export function fingerprintToString(fingerprint: unknown) {
     }
     return JSON.stringify(fingerprint);
 }
+
+export function timeFromNow(date: number) {
```

Comment:
What do you think about using the existing `luxon` library and `Duration` to perform the calculations?
Something like what they were doing in this comment but without the `Intl` dependency
https://github.com/moment/luxon/issues/274#issuecomment-729126515

Reply:
@willr3 Our goal is to handle `lock` file and any dependency that is not providing significant or  less use so we don't need conflicts any more.

Reply:
I thought we were removing `moment` to decrease the build size of the deployed javascript. In either case, I think the proposed change will result in `1 years ago` instead of `1 year ago` (same issue with other increments)